### ISSUE TITLE
Question: Is there the ability for an estate agent to login and save a message against a user?

### ISSUE THREAD

**Issue by tuscan88** [dezrez/DezrezCoreAPI#35]
I can see that there is a chat application in the Rezi web applicant but it appears to only let me message other agents rather than the property vendors. I can also see that when viewing a property there is the option to contact the customer which gives me the option to send an SMS or email to the vendor. Is there a way to simply store a message either against a user or a property though which can then be retrieved via the API?I can see in the API docs there is the api/event/addchatmessage method which says that it adds a message to the property. How does this method tie in with the Rezi web applicant? 

Basically what I need to achieve is the estate agent should be able to login to Rezi and store a message against either a user or one of their properties. Then when the user logs into our website I need the ability to retrieve these messages via the API. If this isn't possible with the existing messaging/chat functionality I'm thinking maybe notes could be used instead to achieve this?

**Comment by marc-harry** [comment]
Hi @geoffs3310,
So the best endpoints to achieve what your looking to do is the `api/event/noteevent/add` and then specifying either RoleId if its to be associated to property marketing role or GroupId for a group of people. 

You can then get these notes back with all other events using `api/group/events` or `api/role/events` or if you just want notes you can use `api/group/notes` or `api/role/notes`.

Cheers,
Marc

**Comment by tuscan88** [comment]
Great thanks
